src/css/components/_side-nav-toggle.scss

```diff
@@ -1,7 +1,8 @@
 .side-nav-toggle {
   cursor: pointer;
-  display: block;
+  display: flex;
   margin-right: 20px;
+  gap: 10px;
   position: relative;
   z-index: 300;
 }
```

Comment:
Side-nav-toggle / previous & next icons in dark mode:

![image](https://github.com/user-attachments/assets/872a13b3-6b47-4fdd-a8fe-a6e004fac215)